clarifies license (#8857)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/billing.md

### What counts as a seat license?
You can learn more about allocating users to your account in [Users and licenses](/docs/cloud/manage-access/seats-and-users).
There are four types of possible seat licenses:

> * **Analyst**\* &mdash; for permission sets assigned and shared amongst those who don't need day-to-day access.

Reviewer: Looks like the `legacy` language was nixed, but maybe we could say "discontinued" instead
```suggestion
* **Analyst**(Discontinued) &mdash; for permission sets assigned and shared amongst those who don't need day-to-day access.
```
or pull the language up to the sentence rather than * it

```suggestion
* **Analyst** &mdash; for permission sets assigned and shared amongst those who don't need day-to-day access. The Analyst license has been discontinued and is no longer available for purchase. Existing accounts with purchased licenses still have access.
```

Author: i'll double check that with legal, thanks Matt!

Author: discussed in thread and not to add

* **Developer** &mdash; for permission sets that require day-to-day interaction with the <Constant name="dbt_platform" />.
* **IT** &mdash; for access to specific features related to account management (for example, configuring git integration).
* **Read-Only** &mdash; for access to view certain documents and reports.

> \* The Analyst license type is not available for new purchase.

Author: ```suggestion
\* The [Analyst license type](/docs/cloud/manage-access/about-user-access?version=1.12#licenses) is not available for new purchase.
```

> \* The Analyst license is not currently available for purchase.

Author: ```suggestion
\* The Analyst license type is not available for new purchase.
```
